modules/core/include/opencv2/core/hal/intrin_sse.hpp

```diff
@@ -1860,6 +1860,11 @@ OPENCV_HAL_IMPL_SSE_LOADSTORE_INTERLEAVE(v_int16x8, short, s16, v_uint16x8, usho
 OPENCV_HAL_IMPL_SSE_LOADSTORE_INTERLEAVE(v_int32x4, int, s32, v_uint32x4, unsigned, u32)
 OPENCV_HAL_IMPL_SSE_LOADSTORE_INTERLEAVE(v_float32x4, float, f32, v_uint32x4, unsigned, u32)
 
+inline v_int32x4 v_cvt_s32(const v_float64x2& a)
+{
+    return v_int32x4(_mm_cvtpd_epi32(a.val));
```

Comment:
No need for define new universal intrinsic, _mm_cvtpd_epi32 already defined in v_round(v_float64x2)